### ISSUE TITLE
Can mean and variance be empty in batchnorm?

### ISSUE THREAD

**Issue by peteryang1** [onnx/onnx#1536]
In tensorflow batchnorm node can have an empty mean and variance input tensor, which means the mean and variance should be determined by the input when running the inference, is it the same in onnx?

tks!

**Comment by peteryang1** [comment]
so "mean" and "variance" input can be empty?
![capture](https://user-images.githubusercontent.com/25981102/47336264-96b0c700-d6c1-11e8-8a92-09abbb92bbdf.PNG)
like this node, mean is a constant tensor which dims is 0.
